apps/proxy/src/app/create-project/create-project.component.html

```diff
@@ -0,0 +1,305 @@
+                    <p>
+                        The Gateway URL is the entry point through which incoming requests are received and forwarded to
+                        the Proxy Server for processing
+                    </p>
+                </ng-container>
+
+                <ng-container *ngTemplateOutlet="gateWayUrl"></ng-container>
+                <div class="d-flex align-items-center justify-content-end">
+                    <button type="button" mat-flat-button color="primary" matStepperNext (click)="checkInputfeild()">
+                        Next
+                        <mat-icon class="material-icons-outlined mat-icon-18 mat-icon-suffix">navigate_next</mat-icon>
+                    </button>
+                </div>
+            </div>
+        </mat-step>
+
+        <mat-step>
+            <div class="d-flex flex-column h-100 mt-3">
+                <ng-container>
+                    <h1 class="fw-bold">Add Destination Url</h1>
+                    <p class="mat-body-2 text-secondary fw-bold">
+                        The destination URL refers to the targeted URL or endpoint where the request will be forwarded
+                        by the proxy Server.
+                    </p>
+                </ng-container>
+                <ng-container *ngTemplateOutlet="desnationUrl"></ng-container>
+                <div class="d-flex align-items-center justify-content-between">
+                    <button type="button" mat-flat-button class="flat-default" matStepperPrevious>
+                        <mat-icon class="material-icons-outlined mat-icon-18 mat-icon-prefix">navigate_before</mat-icon
+                        >Back
+                    </button>
+                    <button mat-raised-button color="accent" (click)="submit()">Submit</button>
+                </div>
+            </div>
+        </mat-step>
+    </mat-stepper>
+</div>
+
+<ng-template #createProject>
+    <div class="my-3 flex-grow-1 mt-5">
+        <form class="w-50">
+            <ng-container
+                *ngTemplateOutlet="
+                    inputField;
+                    context: {
+                        fieldControl: primaryDetails.get('name'),
+                        fieldConfig: {
+                            label: 'Enter Your project name',
+                            is_required: true,
+                            patternErrorText: 'Name must contain only alphanumeric and underscore'
+                        }
+                    }
+                "
+            ></ng-container>
+        </form>
+
+        <form class="w-50">
+            <mat-form-field appearance="outline" class="no-padding w-xs-100">
+                <mat-label>Select Environment</mat-label>
+                <proxy-cdk-scroll scrollableElementId="environments">
+                    <mat-select multiple [formControl]="primaryDetails.get('selectedEnvironmentsControl')">
+                        <mat-select-trigger>
```

Comment:
mat-select automatically show comma separated values and `Select Environment` is displayed via mat-label
you can this in placeholder also
why used `mat-select-trigger` ?